src/GodMode.sol

```diff
@@ -82,33 +82,59 @@ library GodMode {
     }
 
     /// @dev Sets the balance for `who` to `amount` for `token`.
-    /// Note this only works for contracts compiled under Solidity. Vyper contracts use a different storage structure for maps.
-    /// See https://twitter.com/msolomon44/status/1420137730009300992?t=WO2052xM3AzUCL7o7Pfkow&s=19
+    /// Note this only works for contracts compiled under Solidity and Vyper.
```

Comment:
I would just remove this. "only solidity and vyper" I'm not sure what else there is 😅 

Also, can you match the comment for overloaded functions too.

Reply:
haha good point, was just "updating" probably don't have to worry too much about yul or ewasm